### PR TITLE
Fix/refactoring templates and register view

### PR DESCRIPTION
**Pull request by slevvvvin**

### CODE REVIEW

File: blog/static/blog/blog.css [hunk 1]
```diff
@@ -10,4 +10,23 @@
     white-space:nowrap;
     text-overflow:ellipsis;
     width:500px;
+    margin: 10px 0 10px;
+}
+
+.btn-secondary{
+    margin-top:20px;
```
Comment (TinaArts) [inline blog/static/blog/blog.css]: Please add space after ':'

File: blog/templates/base.html [hunk 2]
```diff
@@ -11,29 +15,26 @@
...
-                    <a class="dropdown-item" href="{% url 'category' item.pk %}">
-                        {{ item }}
-                        </a>
-                    {% endfor %}
+                    {% get_category_list %}
                 </div>
               </li>
-            {% endif %}
             {% if user.is_authenticated %}
                 {% if user.is_superuser %}
```
Comment (TinaArts) [inline blog/templates/base.html]: You can use only is_superuser

File: blog/templates/category_list.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<ul>
+    <li>
+        {% for item in category_list %}
+            <a class="dropdown-item" href="{% url 'category' item.pk %}">
+             {{ item }}
+            </a>
+        {% endfor %}
+    </li>
+</ul>
```
Comment (TinaArts) [inline blog/templates/category_list.html]: Please add new line

File: blog/forms.py [hunk 4]
```diff
@@ -12,7 +12,9 @@ class Meta:
                 'placeholder': 'write title here'
             }),
             'title_tag': forms.TextInput(attrs={'class': 'form-control'}),
-            'author': forms.Select(attrs={'class': 'form-control'}),
+            'author': forms.TextInput(attrs={'class': 'form-control',
+                                             'value': '1',
```
Comment (TinaArts) [inline blog/forms.py]: Don't use magic numbers

File: blog/models.py [hunk 5]
```diff
@@ -13,6 +13,11 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse('home')
 
+    def save(self, *args, **kwargs):
+        superuser_id = User.objects.filter(is_superuser=True).values('id')
```
Comment (TinaArts) [inline blog/models.py]: This logic will not work in case when you have more than one superuser

File: blog/templates/add_post.html [hunk 6]
```diff
@@ -1,10 +1,8 @@
 {% extends 'base.html'   %}
 
-
 {% block content %}
 {% if user.is_superuser %}
-    <h1>Add Post</h1>
-    <br/><br/>
+    <div class="text title"><h1>Add Post</h1></div>
```
Comment (TinaArts) [inline blog/templates/add_post.html]: h1 tag can be in new line it will be more readable

File: blog/templates/delete_post.html [hunk 7]
```diff
@@ -1,19 +1,12 @@
...
-    <h1>Delete Post</h1>
-    <br/><br/>
-    <h3>Delete: {{ post.title }}</h3>
-    <br/>
+    <h2>Delete: {{ post.title }}</h2>
     <div class="form-group">
         <form method="POST">
             {% csrf_token %}
-            <strong>Are you sure?</strong><br/><br/>
+            <strong>Are you sure?</strong>
```
Comment (TinaArts) [inline blog/templates/delete_post.html]: Good way is use 'p' tag as wrapper.
FYI:
The HTML Strong Importance Element ('strong') indicates that its contents have strong importance, seriousness, or urgency. Browsers typically render the contents in bold type.

**Review by TinaArts — CHANGES_REQUESTED** [review]

File: blog/static/blog/blog.css (hunk 1 above)
Comment (slevvvvin) [reply blog/static/blog/blog.css]: done

File: blog/templates/add_post.html (hunk 6 above)
Comment (slevvvvin) [reply blog/templates/add_post.html]: done

File: blog/templates/delete_post.html (hunk 7 above)
Comment (slevvvvin) [reply blog/templates/delete_post.html]: done

File: blog/forms.py (hunk 4 above)
Comment (slevvvvin) [reply blog/forms.py]: done

File: blog/templates/category_list.html (hunk 3 above)
Comment (slevvvvin) [reply blog/templates/category_list.html]: done

File: blog/templates/base.html (hunk 2 above)
Comment (slevvvvin) [reply blog/templates/base.html]: i need 2 ifs because i have just "user" which should be able to leave a comment

File: blog/models.py (hunk 5 above)
Comment (slevvvvin) [reply blog/models.py]: done

File: blog/models.py [hunk 8]
```diff
@@ -1,6 +1,7 @@
 from django.db import models
 from django.contrib.auth.models import User
 from django.urls import reverse
+from myDjangoBlog import settings
```
Comment (TinaArts) [inline blog/models.py]: Import settings in right way

**Review by TinaArts — APPROVED** [review]

File: blog/models.py (hunk 8 above)
Comment (slevvvvin) [reply blog/models.py]: done

**Pull request merged**